src/components/Transaction/TransactionCellList/index.tsx

```diff
@@ -1,37 +1,20 @@
-import React, { ReactNode, useState } from 'react'
-import LoadMoreIcon from '../../../assets/transaction_load_more.png'
-import ShowLessIcon from '../../../assets/transaction_show_less.png'
+import React, { ReactNode } from 'react'
+import { Link } from 'react-router-dom'
 import TransactionCellListPanel from './styled'
+import { Transaction } from '../../../http/response/Transaction'
 
 interface TransactionCellListProps {
   data: any[]
```

Comment:
Please give a more meaningful name instead of `data` and give a data type instead of `any[]`